lib/autoproj/daemon/git_api/services/github.rb

```diff
@@ -53,7 +67,35 @@ def initialize(**options)
                     def pull_requests(git_url, **options)
                         exception_adapter do
                             @client.pull_requests(git_url.path, **options).map do |pr|
-                                PullRequest.from_ruby_hash(git_url, pr.to_hash)
+                                mergeable =
+                                    if pr["merged_at"]
+                                        false
```

Comment:
Why would use a different setting in this case?

Reply:
I ... don't know actually. Will remove.